src/api/prompt.ts

```diff
@@ -466,9 +466,11 @@ export function notifyPromptActionRequested(action: "next" | "previous" | "clear
 }
 
 /**
- * Awaits a keypress from the user and returns the entered key.
+ * Awaits for one or more keypresses from the user and returns the entered keys.
+ *
+ * `keyCount`: determines the number of keypresses to wait for (defaults to 1)
```

Comment:
nit, for consistency with the rest of the codebase:

```suggestion
 * @param keyCount Determines the number of keypresses to wait for; defaults to 1.
```